build.zig.zon

```diff
@@ -0,0 +1,11 @@
+.{
+    .name = "webview",
+    .version = "1.0.0",
+    .dependencies = .{},
+    .paths = .{ 
```

Comment:
must include all files except for `.github` folder

Reply:
I think when user use zig fetch, all files and folders will be saved except .github too.. it looks bad, but okay